javascript/app.js

```diff
@@ -0,0 +1,481 @@
+
+//A. Q + A
+// How do we assign a value to a variable?
+// We use const, let, and if at all not possible, var, to declare the name of the variable and add an equal sign to initialize the variable.
+// const newVariable = "";
+
+// How do we change the value of a variable?
+//Name the variable and assign a new value. 
+//const newVariable = "string";
+
+//What is pseudocoding and why should you do it?
+//Writing out what you are going to do before you code. Because the majority of your work is narrating what needs to happen and how you're going to do it, and like 20% is just writing it, maybe even less. It's more important to understand exactly what you're doing so that you can reference that when you get lost in actually writing the code
```

Comment:
yep and bc it helps you remember what one small piece of code is doing and kind of block out the rest while you solve that smaller problem